src/validators/any.ts

```diff
@@ -0,0 +1,17 @@
+import { createValidator, Validator } from '../validator'
+
+export interface AnyValidator extends Validator<any> {
+  
+}
+
+const createAnyValidator = (): AnyValidator => {
+  const validateAny = createValidator<AnyValidator>(
+    (validators, applyValidators) =>
+      (value: any): true =>
+        true
```

Comment:
```suggestion
      (value: any): value is any =>
        applyValidators(value)
```
sonst kann man keine runtime Checks mit `.satisfies(...)` hinzufügen 😢